site/en/docs/extensions/mv3/manifest/activeTab/index.md

```diff
@@ -46,13 +46,13 @@ See the [Page Redder][2] sample extension:
   "browser_action": {
```

Comment:
```suggestion
  "action": {
```

`browser_action` has been replaced by `action` in MV3.

Reply:
Oops!